cpp/tests/io/parquet_chunked_reader_test.cu

```diff
@@ -1409,3 +1409,72 @@ TEST_F(ParquetChunkedReaderInputLimitTest, Mixed)
   constexpr int expected_c[] = {20, 21, 15, 14};
   input_limit_test_read(test_filenames, tbl, 128 * 1024 * 1024, 512 * 1024 * 1024, expected_c);
 }
+
+TEST_F(ParquetChunkedReaderTest, TestChunkedReadOutOfBoundChunks)
+{
+  auto const generate_input = [](int num_rows, bool nullable, bool use_delta) {
```

Comment:
nit: `use_delta` is unused.

Reply:
question: is there any reason for declaring this lambda as `const`, but next lambda is not const? (no variables are captured in both).

Reply:
Thanks for pointing this. Just copy pasted segments from one the tests above. Let me make both `const`.

Reply:
All done!